modules/Bio/EnsEMBL/Compara/PipeConfig/ReindexMembers_conf.pm

```diff
@@ -219,7 +219,8 @@ sub core_pipeline_analyses {
         {   -logic_name => 'create_mlss_ss',
             -module     => 'Bio::EnsEMBL::Compara::RunnableDB::PrepareSpeciesSetsMLSS',
             -parameters => {
-                'whole_method_links'        => [ 'PROTEIN_TREES', 'NC_TREES' ],
+                # 'whole_method_links'        => [ 'PROTEIN_TREES', 'NC_TREES' ],
```

Comment:
It's not necessary to delete this commented code, but neither is it necessary to keep it.
```suggestion
```